encoding/csv.ts

```diff
@@ -27,10 +27,6 @@ export type { ReadOptions } from "./csv/_io.ts";
 const QUOTE = '"';
 export const NEWLINE = "\r\n";
 
-export class StringifyError extends Error {
```

Comment:
I think 'this' StringifyError was not deprecated. (StringifyError in `csv_stringify.ts` was deprecated because that's duplicate of this one)

Reply:
Oh, I see. I'll revert.